src/main/webapp/app/entities/quiz/quiz-exercise.model.ts

```diff
@@ -42,7 +42,7 @@ export class QuizExercise extends Exercise {
     public quizPointStatistic?: QuizPointStatistic;
     public quizQuestions?: QuizQuestion[];
     public status?: QuizStatus;
-    public quizMode?: QuizMode;
+    public quizMode?: QuizMode = QuizMode.INDIVIDUAL; // default value
```

Comment:
If you provide a default value, does this still need to be an optional?

Reply:
Hi @ge65cer, thank you for your feedback.
-  I did try making it non optional but I was getting error as `EntityArraryResponseType of exercise.service` is not assignable to `EntityArraryResponseType of quiz-exercise.service`
-  I was aiming to fix the bug with least few changes as possible, hence I decided to make it optional as such (plus similar thing was already done [here](https://github.com/ls1intum/Artemis/blob/edee8b4bbefbf5f6a3373fe47475c516139a7e1e/src/main/webapp/app/entities/exercise.model.ts#L85) in the code.